__tests__/Unit/Components/standup/standup.test.tsx

```diff
@@ -0,0 +1,131 @@
+import { store } from '@/app/store';
+import { Provider } from 'react-redux';
+import StandUpContainer from '@/components/standup';
+import { fireEvent, screen } from '@testing-library/react';
+import { renderWithRouter } from '@/test_utils/createMockRouter';
+
+describe('Standup', () => {
+    test('renders the missed updates of the user', () => {
+        renderWithRouter(
+            <Provider store={store()}>
+                <StandUpContainer />
+            </Provider>
+        );
+        expect(screen.getByTestId('missed-updates')).toBeInTheDocument();
+    });
+    test('renders the completed inputField', () => {
+        renderWithRouter(
+            <Provider store={store()}>
+                <StandUpContainer />
+            </Provider>
+        );
+        expect(
+            screen.queryByTestId('yesterday-input-update')
```

Comment:
you could use better names for test id (not only here everywhere)

Reply:
fixed